packages/lib-classifier/src/store/TranscriptionReductions/mocks/reducedSubject.js

```diff
@@ -0,0 +1,229 @@
+export default {
+  'reducer': 'optics_line_text_reducer',
+  'low_consensus_lines': 3,
+  'transcribed_lines': 10,
+  'frame0': [
```

Comment:
`frame0` here is an array of reduced lines. Those objects are somewhat different from the data that we derive from them in order to show lines on the page.

Reply:
Following our conversation on Slack, should these reductions be defined in a model?